src/main/java/com/dev/cinema/model/dto/hall/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package com.dev.cinema.model.dto.hall;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
 public class CinemaHallRequestDto {
+    @Min(10)
     private int capacity;
+    @NotNull
```

Comment:
You can add message here - @NotNull (message = "...") - then it will be obvious for user what is wrong...